src/main/java/com/twilio/http/HttpClient.java

```diff
@@ -50,6 +53,10 @@ public Response reliableRequest(final Request request, final int[] retryCodes, i
             // Decrement retries
             retries--;
         }
+
+        last_request = request;
+        last_response = response;
```

Comment:
Also I believe the other classes in this library encapsulate class fields generally so we would make these variables private/protected and add getters but not setters (currently someone could set it if they wanted to but not sure why they would)